app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
Please suggest what you would like to see.  I got these from Aaron (and they don't even cover the `kube.external_ip`:

```
	buf, err := yaml.Marshal(map[string]interface{}{
		"env": env,
		"k8s": map[string]interface{}{"storage-class": map[string]interface{}{"shared": "shared", "persistent": "persistent"}}},
	)
```

and

```
type A struct {
   Env string `yaml:"env"`
   K8s B `yaml:"k8s"`
}

type B struct {
   StorageClass StorageClass `yaml:"storage-class"`
}

type StorageClass struct {
   Shared string `yaml:"shared"`
   Persistent string `yaml:"persistent"`
}

yaml.Marshal(A{ 
  Env: env,
  K8s: B{ StorageClass{ Shared: "shared", Persistent: "persistent" } },
})
```

I consider neither of them an improvement over the literal YAML.